ResearchKit/Common/ORKHealthAnswerFormat.m

```diff
@@ -284,9 +336,9 @@ - (ORKAnswerFormat *)impliedAnswerFormat {
         ORKNumericAnswerFormat *format = nil;
             HKUnit *unit = [self healthKitUserUnit];
         if (_numericAnswerStyle == ORKNumericAnswerStyleDecimal) {
-            format = [ORKNumericAnswerFormat decimalAnswerFormatWithUnit:[unit unitString]];
+            format = [ORKNumericAnswerFormat decimalAnswerFormatWithUnit:[unit localizedUnitString]];
```

Comment:
Are these strings for display to the participant, or eventually uploaded to Bridge as part of the answer? If the former, cool; if the latter, researchers in the past have requested they be in one standard localization so data from all users in Synapse is consistent regardless of locale. I'm guessing from the comment below that you're already aware of this, but just being extra sure.